pkgs/applications/networking/mailreaders/himalaya/default.nix

```
@@ -0,0 +1,40 @@
+{ lib, stdenv, rustPlatform, fetchFromGitHub, openssl, pkgconfig, installShellFiles }:
```

`pkgconfig` is an alias of `pkg-config`.

```suggestion
{ lib, stdenv, rustPlatform, fetchFromGitHub, openssl, pkg-config, installShellFiles }:
```